microsoft-365-ops/3.0/modules/ROOT/pages/microsoft-365-ops-connector-reference.adoc

```diff
@@ -0,0 +1,510 @@
+= Microsoft Dynamics 365 For Operations Connector 3.0 Reference - Mule 4
+:page-aliases: connectors::microsoft/microsoft-365-ops-connector-reference.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Microsoft Dynamics 365 for Finance and Operations (Microsoft Dynamics 365 for Operations Connector) enables integration with Microsoft Dynamics 365 for Operations Web API.
+
+
+[[dynamics-365-for-operations]]
+== Microsoft Dynamics 365 For Operations Configurations
+
+=== Parameters
+
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+|Name | String | The name for this configuration. Connectors reference the configuration with this name. | | x
+| Connection a| * <<Dynamics365ForOperations_OauthClientCredentials, OAuth Client Credentials>> {nbsp}
```

Comment:
Remove {nbsp} tag?

Reply:
These don't actually matter. I do like to remove them just to make the copy cleaner for editing.